lib/galaxy/schema/invocation.py

```diff
@@ -516,10 +525,28 @@ class WorkflowInvocationResponse(Model):
     output_values: Dict[str, Any] = Field(
         default=Required, title="Output values", description="Output values of the workflow invocation."
     )
-    messages: List[InvocationMessageResponseUnion] = Field(
-        default=Required,
-        title="Messages",
-        description="A list of messages about why the invocation did not succeed.",
+
+
+class IndexWorkflowInvocationResponse(WorkflowInvocationBaseResponse):
```

Comment:
This is the element view, the one above is the collection view. Both of these are query parameters that are accepted at different endpoints, so it is better to name them `WorkflowInvocationCollectionView` and `WorkflowInvocationElementView`.

I am porting to pydantic 2 now and have made the same changes that are going to conflict, can I ask you to perhaps take a look at gravity for the moment and see if you can make it work with pydantic 1 and pydantic 2 using a conditional pydantic.v1 import (https://docs.pydantic.dev/latest/migration/#continue-using-pydantic-v1-features) ?

Alternatively you could base your work on my branch, but it'll likely still take some days to pass tests.